src/support/utils/sheet.dedupe.fields.ts

```
@@ -0,0 +1,26 @@
+  return fields
+    .reduce((acc, field) => {
+      if (!keys.has(field.key)) {
+        keys.add(field.key);
+        acc.push(field);
+      }
+      return acc;
+    }, [] as Flatfile.Property[])
+    .map((f) => {
+      if (f.type === "reference") {
+        const out = { ...f, type: "string" as any };
+        delete out.config;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)